java/src/org/openqa/selenium/remote/http/jdk/JdkHttpClient.java

```diff
@@ -150,6 +151,11 @@ public void connectFailed(URI uri, SocketAddress sa, IOException ioe) {
       builder.sslContext(sslContext);
     }
 
+    String version = config.version();
```

Comment:
could be simplified to 
```ofNullable(config.version()).map(Version::valueOf).ifPresent(builder::version)```

Reply:
Needs to be consistent with the rest of the file. Just trying to get a quick fix here.